Competition2018/src/main/java/competition/operator_interface/OperatorCommandMap.java

```diff
@@ -109,4 +107,12 @@ public void setupWristCommands(
         oi.operatorGamepad.getifAvailable(9).whenPressed(calibrate);
         loseCalibration.includeOnSmartDashboard();
     }
+
+    @Inject
+    public void setupAutonomousCommands(OperatorInterface oi, DriveNowhereCommand nowhere,
+            DriveForDistanceCommand drive5Ft) {
+        drive5Ft.setDeltaDistance(60);
+        oi.operatorGamepad.getifAvailable(8).whenPressed(drive5Ft);
```

Comment:
Let's just have both of these commands get added to the SmartDashboard, and skip assigning them to buttons.